swap/src/asb/kraken.rs

```diff
@@ -87,6 +87,11 @@ impl RateService {
                     }
                 };
 
+                // If we encounter a heartbeat we skip it and iterate again
+                if msg.eq(r#"{"event":"heartbeat"}"#) {
```

Comment:
How shit is this API? :sweat_smile: 

Do I see it correctly that heartbeat messages have an `event` field and regular responses don't?
Otherwise we could have just made an enum where we deserialize the response and `match` on it in a strongly typed fashion instead of this string comparison here.

Reply:
yes :(

> > With [9496dce](https://github.com/comit-network/xmr-btc-swap/commit/9496dce91708f6196074097320387c221f48ad10) we skip heartbeat messages that have a certain format in production code.
> > Since we are only interested in the ticker data and know the exact heartbeat message I felt this was the solution to go for (jumping back to the start of the loop upon a specific heartbeat message).
> > We could add a mock service to mimic the behavior for testing, but I am not sure that's worth the effort - what do you think @thomaseizinger ?
> 
> No need for the mock-service IMO.
> 
> What might be useful is a small CLI that uses this code and simply prints out what we get on the other end of the channel. That could be useful in debugging and or extending this piece of code.

Agreed - do you see this as e.g. a separate crate at some point so we can use it as a lib in code but also provide the simple CLI?